### PR TITLE
Feature/container and details testing

### PR DESCRIPTION
**Pull request by CLPena**
## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [ ] Refactoring- no new features
- [x] Testing

## Detailed Description
Unit tests for movie container and movie details components

## Why is this change required? What problem does it solve?
N/A

## Were there any challenges that arose while implementing this feature? If so, how were they addressed?
Had to get on the same page about how to import and export the rootReducer across all testing files

## Files modified:
- src/components/MovieContainer/MovieContainer.js
-  src/components/MovieContainer/MovieContainer.test.js
-  src/components/MovieDetails/MovieDetails.js
- src/components/MovieDetails/MovieDetails.test.js
- src/index.js
-  src/reducers/index.js
- src/reducers/movieRatings.js

### CODE REVIEW

File: src/reducers/movieRatings.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const movieRatings = (state = [], action) => {
+
+  switch (action.type) {
+    case 'SUBMIT_RATING':
+      return [...state, action.rating]
+    default:
+      return state;
+  }
+
+}
```
Comment (TimNguyen21) [inline src/reducers/movieRatings.js:10]: I think we had removed this reducer because we are not using it in the App. Are we using it again?

Comment (megan-venetianer) [reply src/reducers/movieRatings.js:10]: Good catch, Tim. I have this removed on my version as well.

Comment (CLPena) [reply src/reducers/movieRatings.js:10]: Weird. I thought I pulled down and merged the master. I'll go back and fix it.

Comment (CLPena) [reply src/reducers/movieRatings.js:10]: Merging isn't fixing it, so I'm just going to manually delete the file from this branch.

Comment (CLPena) [reply src/reducers/movieRatings.js:10]: Is there anything else weird like this that you all are seeing?

**Comment by hborrelli1** [comment]
Looks good! Nice job on the tests!

File: src/reducers/movieRatings.js (hunk 1 above)
Comment (megan-venetianer) [reply src/reducers/movieRatings.js]: Nope, all else looks good! I'll merge now.

Comment (TimNguyen21) [reply src/reducers/movieRatings.js]: Everything else looks fine. I just notice that one code sneak backed in.  Overall, the testing looks great!

**Pull request merged**